spark/v3.1/spark/src/test/java/com/netease/arctic/spark/SparkTestBase.java

```diff
@@ -64,4 +69,21 @@ public void after() {
     System.out.println("  Test End: " + testName.getMethodName() + ", total cost: " + cost + " ms");
     System.out.println("==================================");
   }
+
+  public void assertDescResult(List<Object[]> rows, List<String> primaryKeys) {
+    boolean primaryKeysBlock = false;
+    List<String> descPrimaryKeys = Lists.newArrayList();
+    for (Object[] row : rows) {
+      if (StringUtils.equalsIgnoreCase("# Primary keys", row[0].toString())) {
+        primaryKeysBlock = true;
+      } else if (StringUtils.startsWith(row[0].toString(), "# ") && primaryKeysBlock) {
+        primaryKeysBlock = false;
+      } else if (primaryKeysBlock){
+        descPrimaryKeys.add(row[0].toString());
+      }
+    }
+
+    Assert.assertEquals(primaryKeys.size(), descPrimaryKeys.size());
+    Assert.assertArrayEquals(primaryKeys.toArray(), descPrimaryKeys.toArray());
```

Comment:
whatif mulit primary key column ?   assert equal using Set<>  and compare one by one